src/clock-weather-card.ts

```diff
@@ -223,6 +226,7 @@ export class ClockWeatherCard extends LitElement {
         <clock-weather-card-today-right-wrap>
           <clock-weather-card-today-right-wrap-top>
             ${this.config.hide_clock ? weatherString : localizedTemp ? `${weatherString}, ${localizedTemp}` : weatherString}
+            ${!this.config.hide_humidity && localizedHumidity ? html`<div>${localizedHumidity} Humidity</div>` : ''}
```

Comment:
"Humidity" must be fetched from translations

Reply:
didn't realize what I was doing there, but I get it now. I fully localized it now.